sbol/property.py

```diff
@@ -226,6 +226,16 @@ def __str__(self):
 
 
 class URIProperty(Property):
+
+    def __init__(self, property_owner, type_uri, lower_bound, upper_bound,
+                 validation_rules, initial_value=None):
+        super().__init__(property_owner, type_uri, lower_bound, upper_bound,
+                         validation_rules)
+        if self._rdf_type not in self._sbol_owner.properties:
+            self._sbol_owner.properties[self._rdf_type] = []
+        if initial_value is not None:
+            self.value = initial_value
+
```

Comment:
Does this actually add any new behavior to the class that isn't already in the super constructor?  Is it necessary?

Just curious.

Reply:
This constructor sets up the storage for the property so that the individual methods don't have to check for that every time they are called.

Additionally, the `initial_value` is not passed to the parent constructor. The setting of the `initial_value` is handled in this constructor _after_ the storage has been established.